index.md

```diff
@@ -46,3 +51,96 @@ metadata:
 spec:
   minimal: false
 ```
+
+Or, create the resource graphically in the OperatorHub UI (only available if
+you installed Cryostat via operator bundle):
+
+<details open>
+  <summary>Installed Operators View</summary>
+  <a href="{{site.url}}/images/installed-operators.png" target="_blank">
+    <img src="{{site.url}}/images/installed-operators.png" alt="Installed Operators View">
+  </a>
+</details>
+<details>
+  <summary>Cryostat Resources Before</summary>
+  <a href="{{site.url}}/images/cryostat-resources-before.png" target="_blank">
+    <img src="{{site.url}}/images/cryostat-resources-before.png" alt="Cryostat Resources Before">
+  </a>
+</details>
+<details>
+  <summary>Cryostat Resource Creation</summary>
+  <a href="{{site.url}}/images/cryostat-resource-creation.png" target="_blank">
+    <img src="{{site.url}}/images/cryostat-resource-creation.png" alt="Cryostat Resource Creation Wizard">
+  </a>
+</details>
+<details>
+  <summary>Cryostat Resources After</summary>
+  <a href="{{site.url}}/images/cryostat-resources-after.png" target="_blank">
+    <img src="{{site.url}}/images/cryostat-resources-after.png" alt="Cryostat Resources After">
+  </a>
+</details>
+
+### Deploy an Application
+For demo purposes, let's go ahead and deploy a sample application to our
+OpenShift cluster in the same namespace as our Cryostat instance. If you have
+deployed Cryostat into a namespace where you are already running other
+applications, feel free to continue to the next step.
+
+```bash
+$ oc new-app --docker-image=quay.io/andrewazores/quarkus-test:0.0.2
+$ oc patch svc/quarkus-test -p '{"spec":{"$setElementOrder/ports":[{"port":9096},{"port":9999}],"ports":[{"name":"jfr-jmx","port":9096}]}}'
+```
+
+This is a Quarkus container in JVM mode with JMX enabled and pre-configured to
+listen on port 9096.  After deploying the container we patch its service to
+name the 9096 service port `jfr-jmx`, which Cryostat will detect and use to
+determine that this is a compatible application that it should pay attention to.
```

Comment:
s|compatible application that it should pay attention to.| compatible Java application that it should monitor.